dbfit-java/build.gradle

```diff
@@ -62,7 +72,7 @@ subprojects {
         testCompile "org.hamcrest:java-hamcrest:2.0.0.0"
         testCompile "commons-io:commons-io:2.4"
         testRuntime 'com.dbdeploy:dbdeploy-ant:3.0M3'
-        integrationTestCompile "dummy:fitnesse-standalone:${fitNesseVersion}@jar"
+        integrationTestCompile "org.fitnesse:fitnesse:${fitNesseVersion}:standalone@jar"
```

Comment:
Regarding FitNesse dev snapshots - if we want to pin our dependencies to a specific FitNesse dev snapshot then I'm not sure that this is supported. I'd have to play with it. A release version would be better if possible.